make/autoconf/boot-jdk.m4

```diff
@@ -602,11 +602,12 @@ AC_DEFUN([BOOTJDK_SETUP_BUILD_JDK],
   BUILD_JDK_FOUND="no"
   if test "x$with_build_jdk" != "x"; then
     BOOTJDK_CHECK_BUILD_JDK([
-       if test "x$with_build_jdk" != x; then
-         BUILD_JDK=$with_build_jdk
-         BUILD_JDK_FOUND=maybe
-         AC_MSG_NOTICE([Found potential Build JDK using configure arguments])
-       fi])
+      if test "x$with_build_jdk" != x; then
+        BUILD_JDK=$with_build_jdk
+        BUILD_JDK_FOUND=maybe
+        AC_MSG_NOTICE([Found potential Build JDK using configure arguments])
+      fi
```

Comment:
And here?

Reply:
Actually, I think this is correct. The block is logically indented inside the `([` and `])` start-of-block and end-of-block markers. Kind of the same reasons as all m4 macros (as created by `AC_DEFUN`) is indented two spaces, and not four.

We have precedence for this pattern elsewhere as well, e.g. at line 240 in this file.